index.js

```diff
@@ -87,7 +87,9 @@ module.exports = function serialize(obj, options) {
     // Replace unsafe HTML and invalid JavaScript line terminator chars with
     // their safe Unicode char counterpart. This _must_ happen before the
     // regexps and functions are serialized and added back to the string.
-    str = str.replace(UNSAFE_CHARS_REGEXP, escapeUnsafeChars);
+    if(options.unsafe !== true ) {
```

Comment:
Doesn't this turn it off by default since `options.unsafe` will not exist? Can you make the default to `true` so we do not break existing use?

Also add a space after the `if`.

Reply:
Thanks for the review, will add that space.

The unsafe flag can be anything. Meaning replacement only gets skipped when `unsafe` is explicitly set to the the Boolean true. (=== true) 

I should write a test to prove the new behavour, and do you want me to update the readme while I’m at it?

Reply:
Yes, please add a test for the option and update the options section of the readme.

Reply:
Done!